src/main/java/mate/academy/hibernate/relations/dao/impl/MovieDaoImpl.java

```diff
@@ -12,11 +16,37 @@ public MovieDaoImpl(SessionFactory sessionFactory) {
 
     @Override
     public Movie add(Movie movie) {
-        return null;
+        SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
```

Comment:
You should not explicitly get the SessionFactory from HibernateUtil here. Instead, use the sessionFactory provided by the constructor of this class.